src/Core/Storage.fs

```diff
@@ -71,13 +71,13 @@ let buildQuery<'T when 'T : (new : unit -> 'T) and 'T :> ITableEntity> filters =
         |> Seq.reduce combineFilters
         |> TableQuery<'T>().Where
 
-let tryGetRandom<'T when 'T : (new : unit -> 'T) and 'T :> ITableEntity> tableName filters =
+let tryGetRandom<'T when 'T : (new : unit -> 'T) and 'T :> ITableEntity> tableName azureFilters postFilters =
```

Comment:
I would create another version of `tryGetRandom` defaulting to `[]` for the last argument. Maybe even two like `tryGetRandomNoPostFilters`, `tryGetRandomNoFilters`. That seem to simplify the code below

Reply:
Yes, good point, updated